Source/Mosa.DeviceSystem/Bitmap.cs

```diff
@@ -9,9 +9,9 @@ public static Image CreateImage(byte[] data)
 			if (data[0] != (byte)'B' || data[1] != (byte)'M')
 				return null;
 
-			var bpp = data[0x1C] / 8;
+			var bpp = (uint)(data[0x1C] / 8);
```

Comment:
Consider changing variable name to bytesPerPixel as bpp usually represents bits per pixel.